<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormFileBrowser</class>
 <widget class="QWidget" name="FormFileBrowser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>572</width>
    <height>511</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="handleWidth">
         <number>5</number>
        </property>
        <widget class="FormFileNavigation" name="wFileNavigation" native="true"/>
        <widget class="QStackedWidget" name="stackedWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="FormPageFiles" name="pageFiles">
          <layout class="QGridLayout" name="gridLayout_3">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="FormPageFilterdFiles" name="pageFilteredFiles">
          <layout class="QGridLayout" name="gridLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <property name="spacing">
            <number>0</number>
           </property>
          </layout>
         </widget>
         <widget class="FormPageProgressInfo" name="pageProgressInfo">
          <layout class="QGridLayout" name="gridLayout_5"/>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>FormFileNavigation</class>
   <extends>QWidget</extends>
   <header location="global">formfilenavigation.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormPageProgressInfo</class>
   <extends>QWidget</extends>
   <header>formpageprogressinfo.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormPageFilterdFiles</class>
   <extends>QWidget</extends>
   <header>formpagefilterdfiles.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FormPageFiles</class>
   <extends>QWidget</extends>
   <header>formpagefiles.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
